TripKitUI/controller/TKAlertViewController.swift

```diff
@@ -41,40 +88,32 @@ public class TKAlertViewController: UITableViewController {
     }
   }
   
-  public var transitAlerts: Observable<[TKAlertModel]>?
-  @objc public weak var alertControllerDelegate: TKAlertViewControllerDelegate?
-  
   // MARK: - View lifecycle
   
   override public func viewDidLoad() {
     super.viewDidLoad()
     
     self.title = Loc.Alerts
     
-    if let navigator = navigationController,
-      let topCtr = navigator.viewControllers.first, topCtr == self {
-      let doneButton = UIBarButtonItem(barButtonSystemItem: .done, target: nil, action: nil)
-      doneButton.rx.tap
-        .subscribe(onNext: { [unowned self] in
-          self.dismiss(animated: true, completion: nil)
-        })
-        .disposed(by: disposeBag)
-      navigationItem.leftBarButtonItem = doneButton
-    }
-    
     tableView.rowHeight = UITableViewAutomaticDimension
     tableView.estimatedRowHeight = 150
-    tableView.register(TKAlertCell.nib, forCellReuseIdentifier: String(describing: TKAlertCell.self))
+//    tableView.register(TKAlertCell.nib, forCellReuseIdentifier: String(describing: TKAlertCell.self))
+    tableView.register(UINib(nibName: "TKNewAlertCell", bundle: Bundle(for: TKNewAlertCell.self)), forCellReuseIdentifier: "TKNewAlertCell")
```

Comment:
Before you merge this, please get rid off the old one and name the new one just `TKAlertCell`

Reply:
Done